content/2020/_index.md

```diff
@@ -17,10 +17,11 @@ layout: "single"
 {{< grid/section-container id="registration" containerClass="backdrop padding-top-40 padding-bottom-40" >}}
   {{< grid/div class="container" isMarkdown="false" >}}
     {{< events/registration year="2020" >}}
-JakartaOne Livestream is a one-day virtual conference for developers and technical business leaders that brings insights into the current state and future of Jakarta&reg; EE and related technologies focused on developing cloud native Java applications.
+Thank You for making the first JakartaOne Livestream in Spanish event a great success!
```

Comment:
This line doesn't seem accurate as this is the 2020 event, not the Spanish event

Reply:
Thanks! Nice catch! I copied that text and forgot to copy the new one > <

Reply:
But I think I need to investigate why "end_text" is not in use. The logic is there. 🤔